upgrade/1.5.1/index.md

```diff
@@ -0,0 +1,302 @@
+# CSM 1.5.1 Patch Installation Instructions
+
+* [Introduction](#introduction)
+* [Bug fixes and improvements](#bug-fixes-and-improvements)
+* [Steps](#steps)
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management `v1.5.1` from `1.5.0`. 
+If upgrading from CSM `v1.4.x` directly to `v1.5.1`, follow the procedures described
+in [Upgrade CSM](../README.md) instead.
+
+## Bug fixes and improvements
+
```

Comment:
@ndavidson-hpe how do we communicate your hot fix that went into 1.5.0 for CASMINST-6715 is now in this release?  What problem does it actually fix?

Reply:
It fixes an issue where an old version of the spire-agent was installed to Nexus.

Reply:
but what text should be  used to communicate this to the customer? Link the ticket? Link the hot fix information?

Reply:
Probably a link to the hotfix information. Not sure how much customer can see internal triage tickets.

Reply:
@ndavidson-hpe did you come up with text on what can be added here?

Reply:
Made a comment with a suggested change to add the line for the spire-agent fix.